### ISSUE TITLE
Not able to put focus cursor on inoreader.com search field

### ISSUE THREAD

**Issue by jaredbidlow** [brookhong/Surfingkeys#196]
I cannot, whether though surfingkeys hints or by manually clicking, enter the "Search of Subscribe" field on inoreader.com

Seems to be an issue only when I have surfingkeys enabled

**Comment by axs221** [comment]
@jaredbidlow A few thoughts here. 

Do you want to use Surfingkeys on inoreader.com? You can press Alt+s to disable on just that site. That is what I would do myself as inoreader.com has plenty of good hotkeys, many Vim-based and many that conflict (such as j/k to navigate).

If you do want to use it on inoreader, do you care for globally binding the forward slash ("/") key to search in Surfingkeys? I am able to search in inoreader if I disable the binding in Surfingkeys settings. inoreader uses forward slash to enter into that box.

> unbind('/')

Also, I was able to reproduce the Surfingkeys hints problem, but I wasn't able to reproduce it by manually clicking: manually clicking worked fine for me. When reproducing using hints ("f" key) I saw it stutter (enter the box, and then immediately act as if I escaped the box). I am assuming this is what you are seeing as well:

https://cl.ly/081P1s023B26

**Comment by jaredbidlow** [comment]
I'm not looking to disable any of my sk config on this website. I often scroll through list view with sk bindings and prefer it to to j/k. I've used inoreader for about two years and I have complaints about how the navigation is not effortless. however, im looking into how to tweak Surfingkeyd config to allow for
-filtered hints in inoreader as there are too many
-focus article at top or after scrolling so the space bar expands the top article and not one outside of the current view

I've been able to use a leader key to access inoreaders default shortcuts ('g.' brings up the jumper, for example)
